src/internal/operations/backup.go

```diff
@@ -201,8 +201,8 @@ func (op *BackupOperation) Run(ctx context.Context) (err error) {
 		return opStats.writeErr
 	}
 
-	// TODO: should always be 1, since backups are 1:1 with resourceOwners now.
-	opStats.resourceCount = len(data.ResourceOwnerSet(cs))
+	// should always be 1, since backups are 1:1 with resourceOwners.
+	opStats.resourceCount++
```

Comment:
nit: is there ever a time we'll need to call `BackupOp.Run()` multiple times and have resourceCount > 1? Otherwise it seems easier to grok if it's just `opStats.resourceCount = 1` since readers won't need to know the previous value of `opStats.resourceCount`

Reply:
Either way works for now.  We should probably clean up that tracking to match current behavior in general.